frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx

```diff
@@ -275,7 +317,9 @@ const InfraEnvsTable: React.FC<InfraEnvsTableProps> = ({ infraEnvs, agents }) =>
                                     },
                                 ],
                                 keyFn: (infraEnv: CIM.InfraEnvK8sResource) => infraEnv.metadata.uid as string,
-                                actionFn: deleteInfraEnv,
+                                actionFn: (infraEnv: CIM.InfraEnvK8sResource) => {
```

Comment:
In this table action, in the table of infraenvs in the delete modal dialog, instead of namespace column it would be good to display agents count [1] and 'Cannot be deleted` [2] next to it if the infraenv has some agents. We're using this in the cluster mass deletion as well.

[1]https://github.com/stolostron/console/blob/main/frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx#L177-L185
[2]https://github.com/openshift-assisted/assisted-ui-lib/blob/master/src/cim/components/modals/MassDeleteAgentModal.tsx#L67-L92

Reply:
Changed!